src/inet.c

```diff
@@ -49,6 +49,9 @@ int uv_inet_ntop(int af, const void* src, char* dst, size_t size) {
   /* NOTREACHED */
 }
 
+char* uv_if_indextoname(unsigned int ifindex, char* ifname) {
+    return if_indextoname(ifindex, ifname);
```

Comment:
@saghul and @santigimeno I have read the contribution instructions a few times.  The problem is that some of the related functions are already in `src/inet.c`, and some of them are in `src/uv-common.c`.  None of them are so far in `src/unix/` or `src/win`.  Hence, in `src/unix/` or `src/win` there are basically no existing files where I could add this function.  I would not think the right choice is to create yet another source file for just this simple function.

So, since there seems to be already some `#ifdef _WIN32` conditionals in `src/uv-common.c`, and since all Unix variants (including Linux) should have the same underlying function signature anyway, I'm now moving the source code from `src/inet.c` (where it would belong IMHO) to `src/uv-common.c`, with the actual implementation behind `#ifdef _WIN32`.  Personally, I find the solution ugly due to the new `#ifdef`, but I couldn't find any better place.